docs/unit-tests.md

```diff
@@ -15,5 +15,5 @@ See [Automated Tests](automated-tests.md) for details on how to set up the tests
 * Where required use the built-in unittest.mock assertions to validate calls - i.e. `mock_object.assert_called_once_with(arg)`
 
 
-[back to Automated Tests](automated-tests.md)  
```

Comment:
the double spaces at the end of this line force a new line when the README is viewed in the browser - so they need to stay and be excluded from pre-commit

Reply:
excluded